lib/charms/opensearch/v0/opensearch_tls.py

```diff
@@ -30,7 +30,7 @@
     generate_csr,
     generate_private_key,
 )
-from ops.charm import ActionEvent, RelationBrokenEvent, RelationJoinedEvent
```

Comment:
Hi @carlcsaposs-canonical thanks for this.

I reviewed TLS lib docstring (most recent one), and I noticed two things:
1) They are also recommending to use `relation-joined`: we need a bug there asking to update the docs in this case
2) I also noticed they have more events that we should track in opensearch - the [invalidated events](https://github.com/canonical/self-signed-certificates-operator/blob/7f41b9f22feecf55f556c8e80ea7e6a26e646321/lib/charms/tls_certificates_interface/v3/tls_certificates.py#L159-L165)

For (2), I've filed a separate bug here: #204.